.gitattributes

```diff
@@ -1,8 +1,6 @@
-# Auto detect text files and perform LF normalization
-* text=auto
-# Declare files that will always have LF line endings on checkout.
-*.sh text eol=lf
-*.prop text eol=lf
-*.md text eol=lf
-*.xml text eol=lf
-META-INF/** text eol=lf
+# Declare files that will always have LF line endings on checkout.

```

Comment:
I thought we got rid of LF

Reply:
no. lf is line ending in files. lf is linux which android needs and crlf is windows. it has to do with text files/xmls. your thinking of the big file thing i added that we got rid of